index.rst

```diff
@@ -1,171 +1,782 @@
+                <li>Follow the installation instructions</li>
+                <li>Customize as needed</li>
+            </ol>
+            <a href="/projects/" class="btn btn-primary">Browse Projects</a>
+        </div>
+    </div>
 
-.. _documentation:
+Basic Configuration Example
+---------------------------
 
-Documentation
--------------
+Here's a simple example of an ESPHome configuration file:
 
-You will find all :doc:`ESPHome documentation here<components/index>`.
+.. code-block:: yaml
 
-.. _contributing:
+    # Basic configuration for an ESP32 device
+    esphome:
+      name: living_room_sensor
+      friendly_name: Living Room Sensor
 
-Contributing
-------------
+    # Hardware configuration
+    esp32:
+      board: esp32dev
+      framework:
+        type: arduino
 
-ESPHome depends on and welcomes contributions from our community. If you'd like to contribute, please see
-:doc:`Contributing <guides/contributing>`.
+    # Enable Home Assistant API
+    api:
+      encryption:
+        key: "YOUR_ENCRYPTION_KEY"
+
+    # Enable over-the-air updates
+    ota:
+      password: "YOUR_OTA_PASSWORD"
+
+    # Connect to WiFi
+    wifi:
+      ssid: !secret wifi_ssid
+      password: !secret wifi_password
+      
+      # Enable fallback hotspot if WiFi connection fails
+      ap:
+        ssid: "Living Room Sensor Fallback"
+        password: "YOUR_FALLBACK_PASSWORD"
+
+    # Enable web server
+    web_server:
+      port: 80
+
+    # Add a temperature sensor
+    sensor:
+      - platform: dht
+        pin: GPIO15
+        temperature:
+          name: "Living Room Temperature"
+        humidity:
+          name: "Living Room Humidity"
+        update_interval: 60s
+
+.. _using-with-home-assistant:
+
+Using with Home Assistant
+=========================
+
+ESPHome and Home Assistant are designed to work together seamlessly, providing a powerful smart home ecosystem.
+
+Automatic Discovery
+-------------------
+
+Home Assistant automatically discovers ESPHome devices on your network. When you add a new ESPHome device:
+
+1. Make sure your device is connected to the same network as Home Assistant
+2. Home Assistant will show a notification when it discovers the new device
+3. Click "Configure" to add it to your system
+4. The device and all its entities will appear in your Home Assistant dashboard
+
+
+Dashboard Integration
+---------------------
+
+ESPHome entities appear in Home Assistant just like any other device:
+
+* **Sensors** show up in the appropriate cards and graphs
+* **Switches and lights** can be controlled directly
+* **Binary sensors** can trigger automations
+* **Climate devices** integrate with the thermostat controls
+
+Automations
+-----------
+
+You can create powerful automations using ESPHome devices in Home Assistant:
+
+* Use sensor readings to trigger actions
+* Control ESPHome devices based on conditions
+* Include ESPHome devices in scenes and scripts
+* Create complex multi-device automations
```

Comment:
I worry this leads people to think they can get ESPHome devices talking to each other directly...and we're not there just yet. haha
```suggestion
* Create complex automations based on a number of inputs and/or conditions
```

Reply:
It does say "automations using ESPHome devices _in Home Assistant_"